packager/react-packager/src/DependencyResolver/Package.js

```diff
@@ -90,4 +91,10 @@ class Package {
   }
 }
 
+function getReplacements(pkg) {
+  return pkg['react-native'] == null
```

Comment:
If Browserify supports it and it's not too hard to add, we could support it eventually, if the need arises.